README.md

```diff
@@ -34,6 +34,13 @@ Compare functions output the following:
 
 ```
 
+## Install
+
+```sh
```

Comment:
Hi @bkowshik , was just checking https://help.github.com/articles/creating-and-highlighting-code-blocks/ which helps in highlighting the code. But mistakenly committed it while previewing.